README.md

```diff
@@ -1,2 +1,250 @@
-# Hello_Microverse
-My first Microverse Project
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
```

Comment:
- [ ] Keeping commented code in your codebase makes your work look clustered. Kindly delete such to have a clean and professional looking code base 👍